CMakeLists.txt

```diff
@@ -1250,6 +1250,16 @@ if(USE_MIMALLOC)
   include_directories(third_party/mimalloc/include)
 endif()
 
+if(USE_MKLDNN_ACL)
+    find_package(ACL REQUIRED)
+    if(ACL_FOUND)
+        include_directories(${ACL_INCLUDE_DIRS})
```

Comment:
This is a really bad pattern, what's wrong with `target_include_directories(torch_cpu PRIVATE $(ACL_INCLUDE_DIRS})`?

Reply:
Ack, I removed this from the root `CMakeLists.txt` into the caffe2 one where `torch_cpu` is defined.
I removed the redundant `if(ACL_FOUND)` check and used `target_include_directories(torch_cpu PRIVATE $(ACL_INCLUDE_DIRS})` instrad of `include_directories(${ACL_INCLUDE_DIRS})` as you suggested.

Does it look better now?